CMakeLists.txt

```diff
@@ -161,8 +164,10 @@ if (NOT OPTION_DEDICATED)
     link_package(Allegro)
     link_package(FREETYPE TARGET Freetype::Freetype)
     link_package(Fontconfig TARGET Fontconfig::Fontconfig)
-    link_package(ICU_lx)
-    link_package(ICU_i18n)
+    if (NOT APPLE)
+        link_package(ICU_lx)
+        link_package(ICU_i18n)
+    endif (NOT APPLE)
```

Comment:
No need for `NOT APPLE` here, the package is not found anyway

Reply:
Well, without this change, the package is found if present; I just checked and `icu-lx` does indeed get linked.

Reply:
But find_package() (line 38) is guarded so it's not even searched. Be sure to always clear the cache when testing modifications in cmake files.

Reply:
Ah, I see — I thought you were referring to the _other_, vaguely similar line. I've removed this one!